### PR TITLE
[MMDS][Cloud][UI] Fix language in MMDS and Profiles in pipelines

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Modifies references to `GCE DataProc` to `Google Cloud Dataproc` for profiles
- Modifies labels for fields while creating a profile in UI
- Modifies title of individual steps while creating an experiment/model in MMDS in UI.

### CODE REVIEW

File: cdap-ui/app/cdap/components/Cloud/Profiles/CreateView/index.js [hunk 1]
```diff
@@ -257,7 +257,7 @@ class ProfileCreateView extends Component {
       <Provider store={CreateProfileStore}>
         <div className="profile-create-view">
           <EntityTopPanel
-            title="Create a Google Dataproc Profile"
+            title="Create a profile for Google Cloud Dataproc"
```
Comment (albertshau) [inline cdap-ui/app/cdap/components/Cloud/Profiles/CreateView/index.js]: How come 'Google Cloud Dataproc' is hardcoded in the UI? Is there something missing from the backend?

Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/Cloud/Profiles/CreateView/index.js]: Good point. Now that we get the provisioner information from backend. Have updated.

**Comment by tonybach** [comment]
UI changes LGTM :+1:

**Review by albertshau — APPROVED** [review]
terminology changes lgtm

**Review by bdmogal — APPROVED** [review]

**Pull request merged**